tools/generators/files_and_groups/test/ElementCreator/CreateGroupTests.swift

```diff
@@ -0,0 +1,200 @@
+import CustomDump
+import PBXProj
+import XCTest
+
+@testable import files_and_groups
+
+final class CreateGroupTests: XCTestCase {
```

Comment:
These tests, and the ones for the other group based functions, are a bit much to read, because they are threading values between multiple functions. I'll see if there is a way to refactor the test and/or function to make it more readable.

Reply:
I'm going to leave them for now.